samples/storage_csm_powerflex_v2101.yaml

```diff
@@ -190,26 +195,31 @@ spec:
     # Authorization: enable csm-authorization for RBAC
     - name: authorization
       # enable: Enable/Disable csm-authorization
+      # Default value: false
       enabled: false
       configVersion: v1.10.1
       components:
         - name: karavi-authorization-proxy
           image: docker.io/dellemc/csm-authorization-sidecar:v1.10.1
           envs:
             # proxyHost: hostname of the csm-authorization server
+            # Default value: none
```

Comment:
If we dont provide any value, will it take empty for the host name?

Reply:
If we provide no value to it, it will throw an error(`PROXY_HOST for authorization is empty`) but there is no default value.